test/Debugger/JsDiagGetStackProperties.js.dbg.baseline

```diff
@@ -3,6 +3,7 @@
     "this": "Object {...}",
     "{exception}": "number 1",
     "locals": {
+      "console": "Object {...}",

```

Comment:
Yep, that's why I haven't merged yet. There are some other failures I can't figure out, too.